src/main/java/software/amazon/awssdk/crt/s3/S3MetaRequestOptions.java

```diff
@@ -163,6 +174,24 @@ public HttpRequest getHttpRequest() {
         return httpRequest;
     }
 
+    /**
+     * If set, this file is sent as the request's body.
+     *
+     * When uploading a file, you should set this for better performance than if you
+     * set a body stream on {@link withHttpRequest}).
+     *
+     * @param requestFilePath path to file to send as the request's body.
+     * @return this
```

Comment:
trivial: I think we should mention that if the user sets both, the body stream will be ignored.